packages/x-date-pickers-pro/src/DateRangeCalendar/useDragRange.ts

```diff
@@ -173,10 +167,19 @@ const useDragRangeEvents = <TDate>({
 
   const handleTouchMove = useEventCallback((event: React.TouchEvent<HTMLButtonElement>) => {
     const target = resolveElementFromTouch(event);
-    if (!isDragging || !target) {
+    if (!target) {
       return;
     }
 
+    event.stopPropagation();
```

Comment:
Could you double-check if this is actually needed? 🤔 
Stopping event propagation is evil and dangerous. 🙈 

Based on a quick check, it did not seem to impact the behavior. 🤷